src/openrct2/actions/RideEntranceExitPlaceAction.hpp

```diff
@@ -200,12 +200,12 @@ DEFINE_GAME_ACTION(RideEntranceExitPlaceAction, GAME_COMMAND_PLACE_RIDE_ENTRANCE
         if (_isExit)
         {
             ride_set_exit_location(
-                ride, _stationNum, TileCoordsXYZD(CoordsXYZD{ _loc, z, (uint8_t)tileElement->GetDirection() }));
+                ride, _stationNum, TileCoordsXYZD(CoordsXYZD{ _loc, z, static_cast<uint8_t>(tileElement->GetDirection()) }));
```

Comment:
Cast should not be necessary.

```suggestion
                ride, _stationNum, TileCoordsXYZD(CoordsXYZD{ _loc, z, tileElement->GetDirection() }));
```